android/libraries/rib-base-test/src/main/java/com/badoo/common/ribs/RibsRule.kt

```diff
@@ -0,0 +1,29 @@
+package com.badoo.common.ribs
+
+
+import android.support.test.rule.ActivityTestRule
+import com.badoo.ribs.RibTestActivity
+import com.badoo.ribs.core.Node
+import org.junit.runner.Description
+import org.junit.runners.model.Statement
+
+open class RibsRule(
```

Comment:
For me it doesn't looks flexible to use.
1. I assume, that we want to have one RibsRule on the screen to launch tests.

2. In such case we can't start it manually providing some custom parameters:
For example imagine such test:
```Kotlin
@Test
fun openScreen_titleShown() {
    ribsRule.start { buildRib(title = "Title") }

    screen.title.checks.withText("Title")
}
```

I believe that we should add ability to start Activity manually to be able easily providing some different data.

Reply:
I agree with @Nublo 
However, we can just add a parameter to the rule.
```
val rule = RibsRule(launch = false) {
    build()
}

@Test
fun openScreen_titleShown() {
    ribsRule.launch()

    screen.title.checks.withText("Title")
}

Reply:
Yes, probably we should add a way to launch activity manually. We can add a boolean flag to the constructor with default value and create `start` method.

Reply:
@Nublo Having flag to not launch Activity automatically is fine. But do you actually need to create a different RIB per test? I'm referring to this part:

> `ribsRule.start{ buildRib(title = "Title") }`

We also don't even have this case in our current codebase, and the scope of this PR was only to update existing approach to be compatible with new classes / interfaces. I'd skip it for now.